device/iot-device-samples/multiplexing-sample/src/main/java/samples/com/microsoft/azure/sdk/iot/MultiplexClientManager.java

```diff
@@ -0,0 +1,47 @@
+package samples.com.microsoft.azure.sdk.iot;
+
+import com.microsoft.azure.sdk.iot.device.IotHubConnectionStatusChangeCallback;
+import com.microsoft.azure.sdk.iot.device.MultiplexingClient;
+import lombok.experimental.Delegate;
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.IOException;
+
+@Slf4j
+public class MultiplexClientManager extends ClientManagerBase {
```

Comment:
It's probably worthwhile to add a header to this class that explains why it is worth using, and what it does

Reply:
Since this class is in the samples folder, I am guessing this class (as well as the other managers) are intended as helper classes for the samples, correct?
It would be nice to also mention what benefits one would gain from using such helper classes.

Question: if such helper classes are useful, why can't they be promoted to product code?